src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.hpp

```diff
@@ -40,6 +40,9 @@ class SolidMechanicsLagrangianFEM : public SolverBase
 {
 public:
 
+  /// String used to form the solverName used to register single-physics solvers in CoupledSolver
+  static string coupledSolverAttributePrefix() { return "solid"; }
```

Comment:
We have a catalog name, and this name. Wondering if we want to make the `catalogName()` an aggregate of `coupledSolverAttributePrefix()`?

Reply:
Yes, we could go towards that, maybe in a separate cleanup PR. 

In the current PR, I tried to avoid changing the xml files as much as possible. Unifying this `prefix` (here, `solid`) and the `catalogName` (here, `SolidMechanicsLagrangianSSLE`) may require extensive changes in the input files and changing our convention (lower case for xml attribute, upper case for xml block names). For flow solvers, the `prefix` (`fluid`) that we have used so far does not even appear in the `catalogName`s, so that would require more input file changes. But if this is something that can simplify the parsing of the input by a pre-processor / GUI, maybe we should do it anyway?